openff/toolkit/topology/molecule.py

```diff
@@ -4592,38 +4605,52 @@ def from_mapped_smiles(
     @requires_package("qcelemental")
     def from_qcschema(
         cls: type[FM],
-        qca_record,
-        client=None,
+        qca_object,
         toolkit_registry=GLOBAL_TOOLKIT_REGISTRY,
         allow_undefined_stereo: bool = False,
-    ) -> FM:
+    ):
         """
         Create a Molecule from a QCArchive molecule record or dataset entry
         based on attached cmiles information.
 
-        For a molecule record, a conformer will be set from its geometry.
+        If this method is provided a QCElemental Molecule (or dict representation of a Molecule),
+        it will return a single-conformer OpenFF Molecule.
 
-        For a dataset entry, if a corresponding client instance is provided,
-        the starting geometry for that entry will be used as a conformer.
+        If this method is provided a QCFractal dataset Entry (or dict representation
+        of an Entry), it will return an OpenFF Molecule with at least one conformer, corresponding
+        to the:
+
+        * ``.molecule`` attribute of a single point entry (single conformer)
+        * ``.initial_molecule`` attribute of an optimization or grid optimization entry (single conformer)
+        * ``initial_molecules`` attribute of a torsion drive entry (one or more conformers)
```

Comment:
```suggestion
        * ``.initial_molecules`` attribute of a torsion drive entry (one or more conformers)
```
No preference on whether attributes are referred to by their name or their name with a prepended `.`, but prefer consistency

Reply:
Good catch.